cmd_migrate_db.go

```diff
@@ -205,6 +213,57 @@ func (x *migrateDBCommand) Execute(_ []string) error {
 	// interceptor.
 	ctx := getContext()
 
+	// Before starting any migration, make sure that all existing dbs have
+	// the latest migrations applied so that we do only migrate when all
+	// dbs are up to date. For example it might happen that a user run a
+	// watchtower client earlier and has it switched off since new
+	// migrations have been introduced. In that case we do not start the
+	// migration but first let the user either delete the outdated db or
+	// migrate it by activating the watchtower client again.
+	for _, prefix := range databasesWithMigrations {
+		logger.Infof("Checking db version of database %s", prefix)
+		// Check if the database file exists before trying to open it
```

Comment:
nit: add newline before comment, also missing punctuation in this comment and below.